xray/utils/resultstable.go

```diff
@@ -86,22 +86,54 @@ func splitCves(cves []services.Cve) (string, string, string) {
 	return cve[:len(cve)-1], cvssV2[:len(cvssV2)-1], cvssV3[:len(cvssV3)-1]
 }
 
-func splitComponents(components map[string]services.Component) ([]string, []string, []string) {
-	var compNames, compVersions, compFixedVersions []string
+func splitComponents(components map[string]services.Component) ([]string, []string, []string, []string) {
+	var compNames, compVersions, compFixedVersions, directComponents []string
 	for currCompId, currComp := range components {
 		currCompName, currCompVersion := splitComponentId(currCompId)
 		compNames = append(compNames, currCompName)
 		compVersions = append(compVersions, currCompVersion)
 		compFixedVersions = append(compFixedVersions, strings.Join(currComp.FixedVersions, "\n"))
+		directComponents = append(directComponents, getDirectComponents(currComp.ImpactPaths))
 	}
-	return compNames, compVersions, compFixedVersions
+	return compNames, compVersions, compFixedVersions, directComponents
 }
 
 func splitComponentId(componentId string) (string, string) {
```

Comment:
I suggest that we open a GitHub issue for adding a unit test for thus function.
We can handle this in a seperate PR.

Reply:
Opened an issue:
https://github.com/jfrog/jfrog-cli-core/issues/153